guvnor-ala/guvnor-ala-build-maven/src/main/java/org/guvnor/ala/build/maven/config/MavenBuildConfig.java

```diff
@@ -34,14 +34,14 @@
     */
     default List<String> getGoals() {
         final List<String> result = new ArrayList<>();
-        result.add( "package" );
+        result.add("package");
         return result;
     }
 
     default Properties getProperties() {
         final Properties result = new Properties();
-        result.setProperty("failIfNoTests", "false" );
+        result.setProperty("failIfNoTests",
+                           "false");
```

Comment:
@manstis Thank you for explanation. I'm glad that you mentioned the debugging because now I can imagine there's at least some advantage to choosing this style. It'll be easier for me to accept it if it's actually going to be implemented. 😄

I know there was some discussion but I didn't have the impression that there was a consensus even between those who appear to be in charge of the decision making. So I'm really wondering if this will really be applied globally, including optaplanner repository where the code style is pretty consistent and the change like this will be somewhat intrusive.